src/Symfony/Component/Messenger/Bridge/AmazonSqs/Transport/AmazonSqsReceiver.php

```diff
@@ -43,7 +44,7 @@ public function get(): iterable
         try {
             $sqsEnvelope = $this->connection->get();
         } catch (HttpException $e) {
```

Comment:
I'm not sure if all exception have to be retried... I'm thinking about authentication failure, Invalid Input (ie. payload exceed the maximum size)

Reply:
Good point!